sp800-63b/cover.md

```diff
@@ -3,7 +3,7 @@
 
 # <a name="800-63b"></a> DRAFT NIST Special Publication 800-63B
 
-# Digital Authentication Guideline
+# Digital Identity Guidelines
 
 ### Authentication and Lifecycle Management
```

Comment:
Authenticators are just a piece of this (the part kept by the subscriber), and we're talking about the whole process here (including verifiers, etc.) so I think that might be too narrow.